build/tools/templates/RS.References/RS.References.csproj

```diff
@@ -1,22 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Unintentional deletion? or are these not needed?

Reply:
These were used for RuntimeStore builds (hence the RS prefix) and apparently I forgot to delete them in a previous PR.